manifests/filter/saml_policy.pp

```diff
@@ -0,0 +1,181 @@
+# == Resource: repose::filter::saml_policy
+#
+# This is a resource for generating uri identity filter config files
+#
+# === Parameters
+#
+# [*ensure*]
+# Bool.  Ensure config file present/absent
+# Defaults to <tt>present</tt>
+#
+# [*filename*]
+# String.  Config filename
+# Defaults to <tt>saml-policy.cfg.xml</tt>
+#
+# [*policy_bypass_issuers*]
+# Array, Optional Array of uri's of issuers. 
+# Defaults to <tt>[]</tt>
+#
+# [*keystone_uri*]
+# String, URI of the Keystone instance to use for authentication
+# Defaults to <tt></tt>
+#
+# [*keystone_user*]
+# String, username of the user to authenticate against the keystone_uri for
+# policy acquisition
+# Defaults to <tt></tt>
+#
+# [*keystone_password*]
+# String, password of the user to authenticate against the keystone_uri for
+# policy acquisition
+# Defaults to <tt></tt>
+#
+# [*keystone_pool*]
+# String, name of the pool in http_connection_pool.cfg.xml to use for making
+# requests. Optional, will use default pool if unspecified.
+# Defaults to <tt>undef</tt>
+#
+# [*policy_uri*]
+# String, URI to use for policy acquisition 
+# Defaults to <tt></tt>
+#
+# [*policy_pool*]
+# String, name of the pool in http_connection_pool.cfg.xml to use for policy
+# requests. Optional, will use default pool if unspecified.
+# Defaults to <tt>undef</tt>
+#
+# [*policy_cache_ttl*]
+# Int, Positive integer in seconds for policies time to live in the proxy cache
+# Defaults to <tt>300</tt>
+#
+# [*policy_cache_feed_id*]
+# String, The atom feed id of an atom feed that contains policy events to be
+# used for cache evictions. Policies will be evicted from the cache for all
+# three event types (i.e. CREATE, UPDATE, DELETE). This is optional, if
+# unused cache eviction will be purely off ttl.
+# Defaults to <tt>undef</tt>
+#
+# [*signature_keystore_path*]
+# String, The path to the keystore on the file system that contains the
+# certificates for signing the SAML response.
+# Defaults to <tt>/etc/pki/java/repose-saml-policy.ks</tt>
+#
+# [*signature_keystore_password*]
+# String, The password for using the keystore, defined in
+# `signature_keystore_path`
+# Defaults to <tt></tt>
+#
+# [*signature_keystore_keycert*]
```

Comment:
What do you think about changing this parameter to the following?

signature_keystore_certname